src/reducers/ui.js

```diff
@@ -2,17 +2,21 @@ import { combineReducers } from 'redux'
 
 import {
   FETCH_FILES,
+  FETCH_FILES_FAILURE,
   RECEIVE_FILES,
   CREATE_FOLDER,
   CREATE_FOLDER_SUCCESS,
   UPLOAD_FILE,
-  UPLOAD_FILE_SUCCESS
+  UPLOAD_FILE_SUCCESS,
+  RELOAD_PAGE
 } from '../actions'
 
 const isFetching = (state = false, action) => {
   switch (action.type) {
     case FETCH_FILES:
       return action.folderId
+    case FETCH_FILES_FAILURE:
+      return false
```

Comment:
This line isn't necessary: you can pile up 'case' statements when the instructions to be executed are the same.